File: rest-in-peace-21-1!/rest-in-peace.rb
'''
Created on 22 Apr 2016

@author: RAJESH
https://www.hackerearth.com/problem/algorithm/rest-in-peace-21-1
'''

def solve_problem(n)
	if (n.to_s().include?('21') or n%21 == 0)
		puts 'The streak is broken!'
	else
		puts 'The streak lives still in our heart!'
	end
end

t = gets.chomp().to_i()

if (t < 1 or t > 100)
	exit
end

for i in 0...t
	n = gets.chomp().to_i()

	if (n < 1 or n > 1000000)
		exit
	end

	solve_problem(n)
end

File: rest-in-peace-21-1!/rest-in-peace.py
'''
Created on 22 Apr 2016

@author: RAJESH
https://www.hackerearth.com/problem/algorithm/rest-in-peace-21-1
'''

def solve_problem(N):
    if "21" in str(N) or int(N) % 21 == 0:
        print 'The streak is broken!'
    else:
        print 'The streak lives still in our heart!'
    pass

T = raw_input("")
T = int(T)

if T < 1 or T > 100:
    exit()

for i in range(0, T):
    N = raw_input("")
    N = int(N)
    if N < 1 or N > 1000000:
       exit()

    solve_problem(N)

    